packages/rich-text/src/get-active-object.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Gets the active object, if there is any.
+ *
+ * @param {Object} value Value to inspect.
+ *
+ * @return {?Object} Active object, or undefined.
+ */
+export function getActiveObject( { start, end, objects } ) {
```

Comment:
We have tests for `getActiveFormat`. Should we have tests for `getActiveObject`?